.github/scripts/need-respin.sh

```diff
@@ -0,0 +1,22 @@
+#! /bin/bash
+set -euxo pipefail
+
+image=$1
+#  Get Latest Keycloak release tag
+tag=$(curl -sL http://github.com/keycloak/keycloak/releases/latest -H "Accept: application/json" | jq -r '.tag_name')
+
+docker pull ${image}:${tag}
+
+base_image_url=$(docker inspect ${image}:${tag} | jq -r '.[0].Config.Labels.url')
+
+base_image_name=$(docker inspect ${image}:${tag} | jq -r '.[0].Config.Labels.url' | sed 's/.*#\///' | sed 's/\/images.*//')
+
+docker pull ${base_image_name}:latest
+
+latest_base_image_url=$(docker inspect ${base_image_name}:latest | jq -r '.[0].Config.Labels.url')
+
+if [ "${base_image_url}" = "${latest_base_image_url}" ]; then
```

Comment:
@andreaTP, thanks for submitting those changes. If I understood it correctly, the script will check if the base image had changed to perform the respin on the daily basis.

If that's the case, I'd suggest to respin the build of those images more frequently to fix known CVEs that will eventually emerge.

The latest UBI was released 27 days ago, that means that if there's a new CVE, the only wait to remedite those CVEs is waiting until the next release to benefit from the fix, instead of trigger `microdnf update`. 

Does it make some sense?

//cc @ASzc

Reply:
@andreaTP no problem, we can wait for more feedback. Even with less frequent respins, which is a valid concern. We need some agreement about when to patch those CVEs. As you are already aware, we have 2 alternatives:

1. Patch CVEs only when there's a new UBI image
2. Patch when there are some dependency updates + a new UBI image

Reply:
In theory UBI should be maintained at a Health Score of A at all times, so there shouldn't be significant public CVEs whenever you happen to pull the latest UBI, even though it could be many weeks old (29 days as of this comment, with [no CVEs listed](https://catalog.redhat.com/software/containers/ubi8/5c647760bed8bd28d0e38f9f?container-tabs=security)).

I don't think `microdnf update` would be required at any point to fix a CVE from UBI. Anything that's embargoed wouldn't show up in the updates, and it would be embargoed until a new UBI is published